app/services/migration-notification.js

```diff
@@ -72,9 +73,7 @@ export default Service.extend({
 
   shouldShowWikiRulesAndBlockingPolicyBanner() {
     return (this.wikiVariables.wikiRulesBlockingPolicyBanner
-      && localStorageConnector.getItem(
-        this.wikiRulesBlockingPolicyStorageKey,
-      ) !== this.storageTrueValue);
+            && Cookies.get(this.wikiRulesBlockingPolicyStorageKey) !== this.storageTrueValue);
```

Comment:
I think we should leave previous rule, because when we will release this new version users which already dismissed this message in local storage will see that again.